src/display/api.js

```diff
@@ -1088,16 +1088,24 @@ var PDFWorker = (function PDFWorkerClosure() {
   var nextFakeWorkerId = 0;
 
   function getWorkerSrc() {
+    var suffix;
```

Comment:
> I declared those variables in the top of the scope to avoid hoisting issues, if I just declare them when they are used Javascript it can be confusing.

*For future reference:* Please note that there will be no issues with hoisting when a variable is only being used at *one* level of a function, as was the case here. In this case, the code would actually have been more readable with the variables being declared where they were actually used.